### PR TITLE
Move dlis.types to class attribute

### PR DESCRIPTION
**Pull request by ErlendHaa**

### CODE REVIEW

**Comment by ErlendHaa** [comment]
#106

File: python/tests/test_semantics.py [hunk 1]
```diff
@@ -487,3 +487,53 @@ def test_dynamic_linkage(fpath):
...
+        # parsed normally
+        assert obj.long_name         == longname
+        assert obj.dimension         == [2]
+        assert obj.axis              == [axis]
+
+        # Parameter attributes that's not in Channel should end up in stash
+        assert obj.stash['VALUES']   == [101, 120]
+        assert obj.stash['ZONES']    == [(10, 0, 'ZONE-A')]
+
+def test_remove_objecttype(fpath):
```
Comment (achaikou) [inline python/tests/test_semantics.py]: object_type ?

File: python/dlisio/__init__.py [hunk 2]
```diff
@@ -12,6 +12,74 @@
...
+    ...     self.longname = None
+    ...     self.samples  = []
+
+    Add the new object-type and load the file
+
+    >>> dlisio.dlis.typemap['440-CHANNEL'] = Channel440.create
+    >>> f = dlisio.load('filename')
+
+    Access all objects off type 440-CHANNEL
+    >>> channels440 = f.types['440-CHANNEL']
```
Comment (achaikou) [inline python/dlisio/__init__.py]: New line missing, not rendered

File: python/dlisio/__init__.py [hunk 3]
```diff
@@ -12,6 +12,74 @@
...
+    Remove object-type PARAMETER. PARAMETER objects will no longer
+    have a specialized parsing routine and will be parsed as Unknown objects
+
+    >>> del dlisio.dlis.typemap['CHANNEL']
+    >>> f = dlisio.load('filename')
+
+    See also
+    --------
+
+    attributes : BasicObject.attributes
```
Comment (achaikou) [inline python/dlisio/__init__.py]: If you want to make it actually a link

plumbing.BasicObject.attributes : attributes

**Comment by achaikou** [comment]
:+1: 
no more comments

**Pull request merged**